src/routes/transactions/entities/swap-order-info.entity.ts

```diff
@@ -7,168 +7,132 @@ import {
   ApiProperty,
   ApiPropertyOptional,
 } from '@nestjs/swagger';
+import { OrderStatus } from '@/domain/swaps/entities/order.entity';
 
 export class TokenInfo {
+  @ApiProperty({ description: 'The token address' })
+  address: `0x${string}`;
+
+  @ApiProperty({ description: 'The token decimals' })
+  decimals: number;
+
   @ApiPropertyOptional({
     type: String,
     nullable: true,
-    description: 'The token logo',
+    description: 'The logo URI for the token',
   })
-  logo: string | null;
+  logoUri: string | null;
+
+  @ApiProperty({ description: 'The token name' })
+  name: string;
 
   @ApiProperty({ description: 'The token symbol' })
   symbol: string;
 
-  @ApiProperty({
-    description: 'The token amount in decimal format',
-  })
-  amount: string;
+  @ApiProperty({ description: 'The token trusted status' })
+  trusted: boolean;
 
-  constructor(args: { logo: string | null; symbol: string; amount: string }) {
-    this.logo = args.logo;
+  constructor(args: {
+    address: `0x${string}`;
+    decimals: number;
+    logoUri: string | null;
+    name: string;
+    symbol: string;
+    trusted: boolean;
```

Comment:
What do you think about aligning these to be more in line with the `Erc20Transfer` entity, e.g.

```suggestion
    tokenAddress: `0x${string}`;
    decimals: number;
    logoUri: string | null;
    tokenName: string;
    tokenSymbol: string;
    trusted: boolean;
```

Reply:
Since the object is of a `Token` I think it unnecessary to prepend the `token` to its properties. If the frontend is already using an entity to represent a token I think we can evaluate reusing it though. @compojoom @usame-algan what's your take on this one?

Reply:
I think the same as @fmrsabino - since the object is already called Token, it is a bit weird if props have a Token prefix.